clang-tools-extra/clang-doc/Representation.h

```diff
@@ -362,6 +362,9 @@ struct FunctionInfo : public SymbolInfo {
   // specializations.
   SmallString<16> FullName;
 
+  // Function Prototype
+  SmallString<256> ProtoType;
```

Comment:
This capitalization might imply that this is a type of proto(type), we also use "prototype" elsewhere.
```suggestion
  SmallString<256> Prototype;
```

Reply:
Yeah, that makes sense.